api/lib/config.js

```diff
@@ -28,14 +32,18 @@ var configPath = nconf.get('config')
 || process.env['TEST_CONFIG']
 || path.join(process.cwd(), 'config.json');
 
+if (!fs.existsSync(configPath)) {
```

Comment:
Ah, that's true. My local copy of ripple-rest was prior to this commit https://github.com/ripple/ripple-rest/commit/df4813f8be8e3ca5a7639258ad552926e4d42fa3 which breaks the tests' being runnable without a config. I think it's still desired to be able to run the tests without a config, `[]` should be default `servers` config for ripple-lib in NODE_ENV=test